Add debian/changelog entries to bump the version to a VERSE-specific one.

--- qemu-7.2+dfsg-orig/debian/changelog	2024-02-06 09:38:06.000000000 -0800
+++ qemu-7.2+dfsg/debian/changelog	2024-06-13 16:45:52.023386289 -0700
@@ -1,3 +1,9 @@
+qemu (1:7.2+dfsg-9999+verse1) bookworm; urgency=medium
+
+  * Initial VERSE OpenSUT build with limited featureset
+
+ -- Stuart Pernsteiner <spernsteiner@galois.com>  Thu, 13 Jun 2024 04:40:23 -0700
+
 qemu (1:7.2+dfsg-7+deb12u5) bookworm; urgency=medium
 
   * +revert-monitor-only-run-coroutine-commands-in-qemu_aio_context.patch
